src/Codec/CBOR/Cuddle/CBOR/Validator.hs

```diff
@@ -111,7 +111,11 @@ data CDDLResult
       String
       -- | Rule we are trying
       Rule
-  deriving (Show)
+  deriving (Show, Eq)
+
+isValid :: CBORTermResult -> Bool
+isValid (CBORTermResult _ Valid {}) = True
+isValid _ = False
```

Comment:
I would give it a better name like `isCBORTermResultValid`

Also it needs to be mentioned in the changelog